src/components/Image/Image.st.css

```diff
@@ -12,29 +12,34 @@
 }
 
 .root {
-  -st-states: preload, loaded;
```

Comment:
I removed the st-states because we don't want to use it anymore... So now I'm passing simply classes